fiftyone/multimodal/decoders/registry.py

```
@@ -0,0 +1,54 @@
+
+def register_decoder(
+    name: str, decoder: MultimodalDecoder
+) -> MultimodalDecoder:
+    """Registers a multimodal decoder by name."""
+
+    with _DECODERS_LOCK:
+        if name in _DECODERS:
+            raise ValueError(
+                f"Decoder {name!r} is already registered: "
+                f"{_DECODERS[name]!r}"
+            )
```

Ruff (0.15.11)

[warning] Avoid specifying long messages outside the exception class

(TRY003)